voxpupuli-puppet-lint-plugins.gemspec

```diff
@@ -11,15 +11,14 @@ Gem::Specification.new do |s|
   s.licenses    = 'AGPL-3.0'
 
   # some of the plugins claim to require Ruby 2.4.0 or newer
-  s.required_ruby_version = '>= 2.4.0'
+  s.required_ruby_version = '>= 2.5.0'
 
   # pull in puppet-lint 2.5.0; required for Ruby 3 support
```

Comment:
This comment is now out of date. Please either keep it in sync (why we need 3.1) or remove it.

Reply:
done

Reply:
Comment is still there.

Reply:
sorry, it got mixed up in the rebase.. fixed it now.